src/screens/ConsentScreen.tsx

```diff
@@ -157,3 +171,24 @@ declare module '@styles' {
 }
 
 export type ConsentScreenStyles = NamedStylesProp<typeof defaultStyles>;
+
+export type CustomConsentScreenProps = {
+  /**
+   * The full Consent and Questionnaire FHIR Resources pertaining to the
+   * consent form. Terms and acceptance text can be derived from here
+   */
+  consentForm:
+    | (Consent & {
+        form: Questionnaire;
+      })[]
+    | undefined;
```

Comment:
Two nits:
- I think there is already a shared `ConsentAndForm` type we should use here.
- I think we should only pass a single instance, not an array. Just grab the first one, like the SDK logic does. Thoughts?

Reply:
>I think we should only pass a single instance, not an array. Just grab the first one, like the SDK logic does. Thoughts?

I can do that, but I feel like it would be nice in case anyone has needs to act on multiple consents. There is a todo here mentioning that the sdk may eventually support it

>   // TODO: If needed, allow for accepting multiple consents in a row.

Reply:
I'd say it's fine to ignore that scenario for now for simplicity. The platform in general does not support multiple consents. Supporting that would be a lot of work.